cfme/tests/infrastructure/test_esx_direct_host.py

```diff
@@ -40,7 +42,7 @@ def pytest_generate_tests(metafunc):
         host = hosts[0]
         creds = credentials[host["credentials"]]
         ip_address = resolve_hostname(host["name"])
-        cred = VMwareProvider.Credential(
+        cred = cfme.base.credential.Credential(
```

Comment:
I wouldn't say it is incorrect  but you're right pycharm made weird change.
I've fixed this import.